apps/meteor/packages/rocketchat-i18n/i18n/en.i18n.json

```
@@ -3088,6 +3088,8 @@
   "List_of_Direct_Messages": "List of Direct Messages",
   "List_view": "List View",
   "Livechat": "Livechat",
+  "Livechat_AdditionalWidgetScripts": "Livechat Widget Additional Scripts. Its possible to specify a CSV list of scripts, for example : https://yourUrl/customScrip1.js,https://yourUrl/customScrip2.js",
```

Separate this into 2:
```suggestion
  "Livechat_WidgetLayoutClasses": "Livechat Widget Additional Scripts",
  "Livechat_WidgetLayoutClasses_Description": "A comma-separated list of scripts, for example: https://yourUrl/customScrip1.js,https://yourUrl/customScrip2.js"

```

Notice how we use the same setting name followed by `_Description`, this is an internal convention and the code will use it automatically, without having you to specify the `i18nDescription` key.

---

Same change should be done to the other setting